lua/barbar/fs.lua

```diff
@@ -0,0 +1,30 @@
+--
+-- fs.lua
+--
+
+
+local fs = {}
+
+--- @param filepath string
+--- @param content string
+--- @param mode string
+--- @return boolean
+function fs.write(filepath, content, mode)
+  mode = mode or 'w'
+  local file, open_err = io.open(filepath, mode)
+  if open_err ~= nil then
+    return false
+  elseif file == nil then
+    return false
+  end
+  file:write(content)
```

Comment:
Missing the check for write error. It might return `true` (i.e. "write succeeded") when it actually failed